pkg/genericapiserver/genericapiserver.go

```diff
@@ -585,25 +585,26 @@ func verifyEtcdServersList(options *options.ServerRunOptions) {
 	}
 }
 
-func verifySecurePort(options *options.ServerRunOptions) {
+func verifySecureAndInsecurePort(options *options.ServerRunOptions) {
```

Comment:
After checking the code, I find that there is already a `verifyServiceNodePort` to verify `--service-node-port-range`. If we use `verifyPorts` instead of `verifySecureAndInsecurePort`, is it easy to distinguish what `verifyServiceNodePort` and `verifyPorts` do by name? Or should we also put what `verifyServiceNodePort` does into `verifyPorts` ?
@lavalamp